src/cloudevents/domains/digital-letters/2025-10-draft/data/digital-letters-print-letter-transitioned-data.schema.yaml

```diff
@@ -0,0 +1,19 @@
+$schema: https://json-schema.org/draft/2020-12/schema
+title: PrintLetterTransitioned Data
+description: Base data of the PrintLetterTransitioned event
+type: object
+additionalProperties: false
+properties:
+  subject:
+    $ref: ../defs/print.schema.yaml#/properties/subject
+  status:
+    $ref: ../defs/print.schema.yaml#/properties/status
+  supplierId:
+    $ref: ../defs/print.schema.yaml#/properties/supplierId
+  time:
+    $ref: ../defs/print.schema.yaml#/properties/time
+required:
+  - subject
```

Comment:
In CCM-13476, A/C 3, the table contains: subject - clientId (senderId in the event) and data.origin.subject (messageReference in the event). do we need an extra field?